eZ/Publish/API/Repository/Tests/SearchServiceAggregationTest.php

```diff
@@ -0,0 +1,886 @@
+use eZ\Publish\API\Repository\Values\Content\Query\Aggregation\Field\DateRangeAggregation;
+use eZ\Publish\API\Repository\Values\Content\Query\Aggregation\Field\DateTimeRangeAggregation;
+use eZ\Publish\API\Repository\Values\Content\Query\Aggregation\Field\FloatRangeAggregation;
+use eZ\Publish\API\Repository\Values\Content\Query\Aggregation\Field\FloatStatsAggregation;
+use eZ\Publish\API\Repository\Values\Content\Query\Aggregation\Field\IntegerRangeAggregation;
+use eZ\Publish\API\Repository\Values\Content\Query\Aggregation\Field\IntegerStatsAggregation;
+use eZ\Publish\API\Repository\Values\Content\Query\Aggregation\Field\KeywordTermAggregation;
+use eZ\Publish\API\Repository\Values\Content\Query\Aggregation\Field\SelectionTermAggregation;
+use eZ\Publish\API\Repository\Values\Content\Query\Aggregation\Field\TimeRangeAggregation;
+use eZ\Publish\API\Repository\Values\Content\Query\Aggregation\LanguageTermAggregation;
+use eZ\Publish\API\Repository\Values\Content\Query\Aggregation\ObjectStateTermAggregation;
+use eZ\Publish\API\Repository\Values\Content\Query\Aggregation\Range;
+use eZ\Publish\API\Repository\Values\Content\Query\Aggregation\RawRangeAggregation;
+use eZ\Publish\API\Repository\Values\Content\Query\Aggregation\RawStatsAggregation;
+use eZ\Publish\API\Repository\Values\Content\Query\Aggregation\RawTermAggregation;
+use eZ\Publish\API\Repository\Values\Content\Query\Aggregation\SectionTermAggregation;
+use eZ\Publish\API\Repository\Values\Content\Query\Aggregation\UserMetadataTermAggregation;
+use eZ\Publish\API\Repository\Values\Content\Query\Aggregation\VisibilityTermAggregation;
+use eZ\Publish\API\Repository\Values\Content\Query\AggregationInterface;
+use eZ\Publish\API\Repository\Values\Content\Query\Criterion\ContentTypeIdentifier;
+use eZ\Publish\API\Repository\Values\Content\Query\Criterion\MatchAll;
+use eZ\Publish\API\Repository\Values\Content\Search\AggregationResult;
+use eZ\Publish\API\Repository\Values\Content\Search\AggregationResult\RangeAggregationResult;
+use eZ\Publish\API\Repository\Values\Content\Search\AggregationResult\RangeAggregationResultEntry;
+use eZ\Publish\API\Repository\Values\Content\Search\AggregationResult\StatsAggregationResult;
+use eZ\Publish\API\Repository\Values\Content\Search\AggregationResult\TermAggregationResult;
+use eZ\Publish\API\Repository\Values\Content\Search\AggregationResult\TermAggregationResultEntry;
+use eZ\Publish\API\Repository\Values\ContentType\ContentType;
+use eZ\Publish\API\Repository\Values\ContentType\FieldDefinitionCreateStruct;
+use eZ\Publish\API\Repository\Values\ObjectState\ObjectState;
+use eZ\Publish\Core\FieldType\Checkbox\Value as CheckboxValue;
+use eZ\Publish\Core\FieldType\Time\Value as TimeValue;
+
+/**
+ * Test case for aggregations in the SearchService.
+ *
+ * @see \eZ\Publish\API\Repository\SearchService
+ * @group integration
+ * @group search
+ * @group aggregations
+ */
+final class SearchServiceAggregationTest extends BaseTest
```

Comment:
the class is quite long, can be split into smaller ones? ex. by TermAggregation, test case etc. 
Smaller classes will be more friendly to maintain in future

Reply:
I agree. I will split this class into smaller once (by Aggregation) in follow up PR.